bootstrap/_rpm_common.sh

```diff
@@ -20,11 +20,15 @@ $tool install -y \
   git-core \
   python \
   python-devel \
-  python-virtualenv \
-  python-devel \
+  python-setuptools \
   gcc \
   dialog \
   augeas-libs \
   openssl-devel \
   libffi-devel \
   ca-certificates \
+
+if ! $tool install -y python-virtualenv ; then
+    easy_install pip
+    pip install virtualenv
```

Comment:
indeed that is the case for CentOS 6 EPEL yum repo is needed and more so Python 2.7 version of pip - for me via IUS Community repo https://community.letsencrypt.org/t/redhat-centos-6-x-users-need-python-2-7/2190

    yum list installed python* -q
    Installed Packages
    python.x86_64                                                                                                              2.6.6-64.el6                                                                                                          @base    
    python-devel.x86_64                                                                                                        2.6.6-64.el6                                                                                                          @base    
    python-ethtool.x86_64                                                                                                      0.6-5.el6                                                                                                             @base    
    python-iniparse.noarch                                                                                                     0.3.1-2.1.el6                                                                                                         installed
    python-iwlib.x86_64                                                                                                        0.1-1.2.el6                                                                                                           @base    
    python-libs.x86_64                                                                                                         2.6.6-64.el6                                                                                                          @base    
    python-psutil.x86_64                                                                                                       0.6.1-1.el6                                                                                                           @epel    
    python-pycurl.x86_64                                                                                                       7.19.0-8.el6                                                                                                          installed
    python-setuptools.noarch                                                                                                   0.6.10-3.el6                                                                                                          @base    
    python-urlgrabber.noarch                                                                                                   3.9.1-9.el6                                                                                                           installed
    python-virtualenv.noarch                                                                                                   1.10.1-1.el6                                                                                                          @epel    
    python27.x86_64                                                                                                            2.7.10-1.ius.centos6                                                                                                  @ius     
    python27-devel.x86_64                                                                                                      2.7.10-1.ius.centos6                                                                                                  @ius     
    python27-libs.x86_64                                                                                                       2.7.10-1.ius.centos6                                                                                                  @ius     
    python27-pip.noarch                                                                                                        7.1.0-1.ius.centos6                                                                                                   @ius     
    python27-setuptools.noarch                                                                                                 18.1-1.ius.centos6                                                                                                    @ius     
    python27-tools.x86_64                                                                                                      2.7.10-1.ius.centos6                                                                                                  @ius     
    python27-virtualenv.noarch                                                                                                 13.1.0-1.ius.centos6                                                                                                  @ius